cfme/tests/cloud_infra_common/test_genealogy.py

```diff
@@ -87,6 +88,10 @@ def _():
             parent = pytest.sel.text(opt).strip()
         assert parent.startswith(small_template), "The parent template not detected!"
     else:
-        vm_crud_ancestors = vm_crud.genealogy.ancestors
+        try:
+            vm_crud_ancestors = vm_crud.genealogy.ancestors
+        except NameError:
+            logger.exception("The parent template not detected!")
```

Comment:
Ah, gotcha! I didn't realize it will log both the exception and the message. :smile: Just learned something new :+1: